### PR TITLE
feat: synchronize configmap and secret to sub clusters

### LINKED ISSUE #158: feat: synchronize configmap and secret to sub clusters
**What would you like to be added**:
synchronize configmap and secret to sub clusters

**Why is this needed**:
synchronize configmap and secret to sub clusters

### PR DESCRIPTION
**Pull request by OrangeBao**
#### What type of PR is this?

feat
#### What does this PR do?

synchronize configmap and secret to sub clusters 
#### Which issue(s) does this PR fix?

Fixes #158 
#### Screenshots (if applicable):

```release-note

```

#### Additional Comments:

```release-note
synchronize configmap and secret to sub clusters 
```

### CODE REVIEW

File: pkg/clustertree/cluster-manager/cluster_controller.go [hunk 1]
```diff
@@ -270,5 +270,19 @@ func (c *ClusterController) setupControllers(m *manager.Manager, cluster *cluste
...
 
+	for i, gvr := range podcontrollers.SYNC_GVRS {
+		demoController := podcontrollers.SyncResourcesReconciler{
+			GroupVersionResource: gvr,
+			Object:               podcontrollers.SYNC_OBJS[i],
+			DynamicRootClient:    c.MasterDynamic,
+			DynamicLeafClient:    clientDynamic,
+			ControllerName:       "async-controller-" + gvr.Resource,
+		}
+		if err = demoController.SetupWithManager(mgr, gvr); err != nil {
```
Comment (xyz2277) [inline pkg/clustertree/cluster-manager/cluster_controller.go:281]: why call it demoController?

Comment (OrangeBao) [reply pkg/clustertree/cluster-manager/cluster_controller.go:281]: ok， i get it！

**Pull request merged**